### PR TITLE
compat: add Purse.io info

### LINKED ISSUE #295: Compatibility details for Purse.io and bcoin
bcoin (https://github.com/bcoin-org/bcoin, https://bcoin.io/) is the backend full node and wallet used by Purse.io. They could be listed together or separately (or just one or the other). bcoin offers a range of options, but Purse has been configured with specific settings. I'm happy to make a PR to this repo with details but I'm not sure if the services explicitly need to be tested by an OpTech associate?

Here's the quick details:

SegWit:

- bcoin and Purse support SegWit in all forms: native, nested, send and receive.
- Purse default receive address is bech32 with option for nested.
- Change address type is either legacy or bech32 depending on wallet `witness` setting, Purse always uses bech32 for change.
- bcoin default wallet type is still legacy until https://github.com/bcoin-org/bcoin/pull/827 is merged.

RBF:

- bcoin and Purse reject RBF transactions by default (they do not enter the mempool or wallet until confirmed) meaning they will "show replaced" but not "show original" assuming the replacement is not itself signaling RBF.
- The wallet by default can not send replaceable transactions or replace RBF transactions.
- Mempool replacement for the bcoin full node is in review: https://github.com/bcoin-org/bcoin/pull/811. After merge we'll begin work on adding RBF options to the wallet although RBF will likely remain rejected by Purse, since deposits aren't credited before confirmation anyway.

Screen shots:

Purse -- illustrating bech32 and nested deposit addresses:

Screen Shot 2019-12-12 at 9 08 56 AM
Screen Shot 2019-12-12 at 9 08 17 AM

bcoin -- from our [API docs](http://bcoin.io/api-docs/) illustrating legacy and SegWit wallet account types with receive, change, and nested addresses:

Screen Shot 2019-12-12 at 9 11 27 AM
Screen Shot 2019-12-12 at 9 12 08 AM

### PR DESCRIPTION
**Pull request by pinheadmz**
Closes #295 

Thanks!

### CODE REVIEW

File: _data/compatibility/purse.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      caption: >
+        Purse deposit addresses are bech32 P2WPKH (native segwit) by default.
+    - image: /img/compatibility/purse/segwit/receive-screen-nested.png
+      caption: >
+        As an option for users with legacy wallets, Purse also accepts deposits
+        to P2SH-wrapped P2WPKH (nested segwit) addresses.
+    - image: /img/compatibility/purse/segwit/send-screen.png
+      caption: >
+        Purse allows withdraws to both legacy and segwit addresses.
+    #- image: /img/compatibility/purse/segwit/send-v1.png
```
Comment (bitschmidty) [inline _data/compatibility/purse.yaml]: Suggesting we hide this for now to be consistent with other tests w/ v1 which we decided to hide for now.

File: _data/compatibility/purse.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    send:
+      bech32: "true"
+      change_bech32: "true"
+      segwit_v1: "Transaction can be created and broadcast, relayed by peers
+      compatible with Bitcoin Core v0.19.0.1 or above."
+      bech32_p2wsh: "true"
+  examples:
+    - image: /img/compatibility/purse/segwit/receive-screen-p2wpkh.png
+      caption: >
+        Purse deposit addresses are bech32 P2WPKH (native segwit) by default.
```
Comment (bitschmidty) [inline _data/compatibility/purse.yaml]: Suggested in a commit to change bech32 and segwit to be lowercase to be consistent with [STYLE.md](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md)

Comment (pinheadmz) [reply _data/compatibility/purse.yaml]: Thanks! Sorry for not RTFM :-/

File: _data/compatibility/purse.yaml (hunk 1 above)
Comment (pinheadmz) [reply _data/compatibility/purse.yaml]: Cool.

**Comment by pinheadmz** [comment]
Thanks for the tweaks, @bitschmidty 
ACK  @ 907d1156b6b9f325d60a355c399de7e94589c652

**Comment by bitschmidty** [comment]
Squashed the commits and merged! Thanks again @pinheadmz !

**Pull request merged**